app/cafe.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+day_today = datetime.date.today()
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError

```

Comment:
You must raise an object of error.
```suggestion
            raise NotVaccinatedError()

```